### ISSUE TITLE
Failures in get_test.go for faas-cli 

### ISSUE THREAD

**Issue by jasonprado** [alexellis/arkade#333]
## Expected Behaviour

Tests should pass when running `make all`

## Current Behaviour

```
$ make all
...
--- FAIL: Test_DownloadFaaSCLIDarwin (0.24s)
    get_test.go:94: want: https://github.com/openfaas/faas-cli/releases/download/0.13.0/faas-cli-darwin, got: https://github.com/openfaas/faas-cli/releases/download/0.13.2/faas-cli-darwin
...
```

## Possible Solution

faas-cli has shipped release 0.13.2. The test should expect that now.

## Steps to Reproduce (for bugs)

1. `make all`

## Context

I see failures in my local environment.

## Your Environment

* What Kubernetes distribution are you using?

N/A

* Operating System and version (e.g. Linux, Windows, MacOS):

```
$ uname -a
Linux b26ed16a6250 4.19.121-linuxkit #1 SMP Tue Dec 1 17:50:32 UTC 2020 x86_64 GNU/Linux
$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"```

* What arkade version is this?

```
$ ./bin/arkade version
            _             _      
  __ _ _ __| | ____ _  __| | ___ 
 / _` | '__| |/ / _` |/ _` |/ _ \
| (_| | |  |   < (_| | (_| |  __/
 \__,_|_|  |_|\_\__,_|\__,_|\___|

Get Kubernetes apps the easy way

Version: 0.7.3-1-gf06a764-dirty
Git Commit: f06a7640614490e40f766dfb55736f7d51494fcb
```

**Comment by jasonprado** [comment]
Working on a fix in #334

**Comment by alexellis** [comment]
Thank you :+1:
